### PR TITLE
📖 [Story sidebar] Removed sidebar from docs

### PR DESCRIPTION
**Pull request by mszylkowski**

### CODE REVIEW

**Review by gmajoulet — COMMENTED** [review]
We should remove no implementation code, or all of it. Removing parts of it doesn't get us super far.
Maybe this PR should be about the documentation only?

File: extensions/amp-story/amp-story.md [hunk 1]
```diff
@@ -493,38 +493,11 @@ Example:
...
-  ...
-</amp-story>
-```
-
 #### Other components usable in AMP stories
 
 The following are other components usable in AMP stories that require some story-specific caveats.
 
 -   [amp-consent](https://amp.dev/documentation/components/amp-consent#prompt-ui-for-stories)
--   [amp-sidebar](https://amp.dev/documentation/components/amp-sidebar#sidebar-for-stories)
```
Comment (gmajoulet) [inline extensions/amp-story/amp-story.md:527]: Why didn't you remove the story part within amp-sidebar.md as well?

Comment (mszylkowski) [reply extensions/amp-story/amp-story.md:527]: Will do

**Review by gmajoulet — APPROVED** [review]
Thanks! Do we have a ticket to remove the sidebar code?

**Comment by mszylkowski** [comment]
We have the I2D/I2R for removing the sidebar #33293

**Pull request merged**